src/net/peacefulcraft/sco/swordskills/utilities/DirectionalUtil.java

```diff
@@ -0,0 +1,112 @@
+package net.peacefulcraft.sco.swordskills.utilities;
+
+import org.bukkit.Location;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
```

Comment:
If we set the event priority below, this is needed too.
```suggestion
import org.bukkit.event.EventHandler;
import org.bukkit.event.EventPriority;
```